pkg/commands/deploy.go

```diff
@@ -68,18 +68,20 @@ func NewRemoveCommand(commonOpts *CommonOptions) *cobra.Command {
 			err = sched.Remove(la, sched.Options{
 				Platform:         opts.clusterPlatform,
 				WaitCompletion:   opts.waitCompletion,
-				RTEConfigData:    commonOpts.RTEConfigData,
+				RTEConfigData:    commonOpts.UpdaterConfigData,
```

Comment:
this was originally meant as RTE-specific data. Does NFD need some specific configuration? if yes, it makes sense to generalize; if not, I'd not generalize it (yet)

Reply:
The ```RTEConfigData``` contains information such as the Exclude-list, which NFD has as well.

Reply:
ack. So yes, let's generalize. Could be worth to rename like `UpdaterSpecificConfig` (also the commandline option should be renamed)